solidity/contracts/converter/BancorConverter.sol

```diff
@@ -237,6 +244,10 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
         totalSupplyGreaterThanZeroOnly
     {
         super.acceptTokenOwnership();
+
+        // sync all reserve balances	
+        for (uint16 i = 0; i < reserveTokens.length; i++)	

```

Comment:
@yudilevi:
No point using `uint16` here (`uint256` is good enough, and probably even slightly better in terms of performance).

Reply:
Does the EVM need to work harder to "limit" it to 16 bits?

Reply:
@yudilevi:
1. The type of array.length is `uint256`
2. AFAIK, it is generally recommended to use `uint256`. Not sure how critical this is for local variables, but I think that we should limit the usage of `uint` types smaller than `uint256` only for when it's absolutely necessary (i.e., packed structures), so that we never have any doubts about possible overflows, etc (even if it is obvious that the total number of reserves will never grow beyond 65535 in this case).

Reply:
updated, though keep in mind reserve/connectorTokenCount functions still return uint16

Reply:
Right... well, we can't really change that one now (without breaking the API)...

Reply:
right